README.md

```diff
@@ -1,2 +1,21 @@
-# documentation
-Official Yield Swap Documentation
+# Introduction
+
+## What is Yield Swap?
+
+Yield Swap is a decentralized exchange that allows users to swap interest rates.
```

Comment:
... that allows users to swap fixed for floating yields.

Reply:
Thank you for the link. I incorporated your feedback and rewrote that section a bit. Note that I moved more detailed information about the payer dynamics to the `payers.md`.